x/estaking/keeper/elys_stake_change.go

```diff
@@ -7,23 +7,43 @@ import (
 )
 
 func (k Keeper) SetElysStakeChange(ctx sdk.Context, addr sdk.AccAddress) {
-	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.ElysStakeChangeKeyPrefix))
-	store.Set([]byte(addr), addr)
+	store := ctx.KVStore(k.storeKey)
+	key := types.GetElysStakeChangeKey(addr)
+	store.Set(key, addr)
 }
 
 func (k Keeper) GetElysStakeChange(ctx sdk.Context, addr sdk.AccAddress) (found bool) {
```

Comment:
this function returns if stake change is present and not the key or value. can we change it's name ?

Reply:
Here the key and value both are the same. So checking key exists does the same. Didn't want to change the function signature where it is used